### DISCUSSION TITLE
Conda : command not found

### DISCUSSION THREAD

**Discussion by sandeepmadana** [biostars/biostar-handbook-issues#216] [General]
Hi,

I have enables WSL and installed Ubuntu cannonical.

As suggested in the online book it typed curl http://data.biostarhandbook.com/install.sh | bash in the terminal. 

`# Biostar Handbook installation complete! is  seen.`

I closed the terminal and restarted the terminal, typed 

`conda activate bioinfo. `

output was conda not found
doctor.py not found

**Comment by ialbert** [comment]
a bit odd to be honest, but see the step by step guide, 

https://www.biostarhandbook.com/step-by-step.html

run the three commands that ask for Bash initialization (step 3)

then close and reopen the terminal

**Comment by sandeepmadana** [comment]
I have tried installing from setp 1. went fine until step 7 where i got the CondaHTTP Error.  How to downgrade the to 4.7.
